jsonnet/kube-prometheus/kube-prometheus.libsonnet

```
@@ -60,7 +61,7 @@ local configMapList = k3.core.v1.configMapList;
       ],
     },
   } +
-  ((import 'kube-prometheus/kube-rbac-proxy/container.libsonnet') {
+  ( (kubeRbacProxyContainer) {
```

It should be possible to get rid of one of the braces.
```suggestion
  (kubeRbacProxyContainer {
```

---

Good point :+1: